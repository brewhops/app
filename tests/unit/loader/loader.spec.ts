import Loader from "@/components/loader.vue";
import { shallowMount } from "@vue/test-utils";

describe("loader.vue", () => {
  it("renders loader component", () => {
    const wrapper = shallowMount(Loader);
    expect(wrapper).toMatchSnapshot();
  });
});
