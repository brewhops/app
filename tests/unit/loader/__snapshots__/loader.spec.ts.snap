// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loader.vue renders loader component 1`] = `
<div class="lds-ring">
  <div></div>
  <div></div>
  <div></div>
  <div></div>
</div>
`;
